static/js/TopicLandingPage/TopicLandingSearch.jsx

```diff
@@ -13,6 +13,13 @@ const getSuggestions = async (input) => {
     const _capitalizeFirstLetter = (text)=> {
         return String(text).charAt(0).toUpperCase() + String(text).slice(1);
     }
+    const _extractDisambiguation = (title) => {
+        const match = title.match(/\((.*?)\)$/);
+        return match ? match[1] : null;
+    };
+    const _removeDisambiguation = (title)=>{
+        return title.replace(/\s*\(.*\)\s*$/, '').trim();
```

Comment:
should be `.*?` as the previous regex. Otherwise this can cause some sticky issues if the * is greedy.